src/RageTimer.h

```diff
@@ -25,7 +25,8 @@ class RageTimer
 
 	static double GetTimeSinceStart( bool bAccurate = true );	// seconds since the program was started
 	static float GetTimeSinceStartFast() { return GetTimeSinceStart(false); }
-	static std::uint64_t GetUsecsSinceStart();
+	static int GetTimeSinceBasic();
```

Comment:
I think renaming to `GetTimeSinceStartSeconds` or something similar is probably better. `Basic` isn't really a great identifier.

Reply:
Good call thanks. I must have overlooked that variable name when pushing this.